pysat/utils/time.py

```diff
@@ -156,7 +229,7 @@ def create_datetime_index(year=None, month=None, day=None, uts=None):
         month : array_like of ints or None
         day : array_like of ints
             for day (default) or day of year (use month=None)
```

Comment:
Change to reflect formatting of subsequent suggestion.

Reply:
I took a pass at an update.